<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Homepage</class>
 <widget class="QWidget" name="Homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FilePathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="ConfirmFileBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>File size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FileSizeLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>PE Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PEtypeLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="archInfoHL">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Architecture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ArchLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="viewsHL">
      <item>
       <widget class="QPushButton" name="sectionsView_bt">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Sections</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ConfirmFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>Homepage</receiver>
   <slot>openFileByLineEditPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sectionsView_bt</sender>
   <signal>clicked()</signal>
   <receiver>Homepage</receiver>
   <slot>displaySectionsView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileByLineEditPath()</slot>
  <slot>displaySectionsView()</slot>
 </slots>
</ui>
